src/test/regress/multi_schedule

```diff
@@ -153,4 +153,4 @@ test: multi_drop_extension
 # ----------
 # multi_schema_support makes sure we can work with tables in schemas other than public with no problem
 # ----------
-test: multi_schema_support
+
```

Comment:
Why did we change the order of multi_schema_support in regression test list? If there is a reason, we should also move the comments.